testing/Makefile

```diff
@@ -0,0 +1,33 @@
+STACK_VERSION ?= 8.6.1
+CURRENT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
```

Comment:
use `$(CURDIR)` instead of calculate it, `CURDIR` point to the folder where the `Makefile` is

```suggestion
```